docs/manual/workflows.rst

```diff
@@ -0,0 +1,98 @@
+.. _manual_workflows:
+
+===============
+Beheerprocessen
+===============
+
+Formulieren worden gemaakt, gepubliceerd, bijgewerkt, vervangen en eventueel
+uitgefaseerd. Hieronder staan de typische beheerprocessen voor formulieren die
+u kunt gebruiken om de levensloop van een formulier zo goed mogelijk te laten
+verlopen.
+
+Essentieel bij het beheer zijn 2 opties:
+
+* **Actief** geeft aan dat een formulier beschikbaar is voor iedereen.
```

Comment:
```suggestion
* **Actief** geeft de globale beschikbaarheid aan.
```
otherwise it reads a bit like "active = always available for everyone", but maintenance mode modifies that, and not active means "not available for anyone".